sdk/spring/azure-spring-cloud-context/src/main/java/com/azure/spring/cloud/autoconfigure/context/AzureContextAutoConfiguration.java

```diff
@@ -65,11 +63,4 @@ public TokenCredential credential(Environment environment) {
                                                    .alternativePrefix(AzureProperties.PREFIX)
                                                    .build();
     }
-
-    @Bean
```

Comment:
Why did you delete this?

Reply:
Because of the he interface `SubscriptionSupplier` is under the `azure-spring-cloud-telemetry` module.

Reply:
I see.